scl-extension/src/main/resources/xsd/SCL_CoMPAS.xsd

```diff
@@ -29,7 +30,7 @@ SPDX-License-Identifier: Apache-2.0
     <!--
     SCL File Type element.
     -->
-    <xs:simpleType name="tCompasSclFileType">
+    <xs:simpleType name="tSclFileType">
```

Comment:
Same here, why is this name Changed? This is now back again to the original name.

Reply:
We have fix this issue and we have normalized all name type like this "tCompas......
[SCL_CoMPAS_20220615.zip](https://github.com/com-pas/compas-core/files/8907311/SCL_CoMPAS_20220615.zip)
"

Reply:
Mistake have been corrected in the new version of the xsd file